common/src/main/java/org/apache/drill/common/config/DrillConfig.java

```diff
@@ -115,19 +124,26 @@ public static DrillConfig forClient() {
    * <p>
    * Configuration values are retrieved as follows:
    * <ul>
-   * <li>Check a single copy of "drill-override.conf". If multiple copies are on the classpath, behavior is
-   * indeterminate.  If a non-null value for overrideFileName is provided, this is utilized instead of drill-override.conf.</li>
-   * <li>Check all copies of "drill-module.conf". Loading order is indeterminate.</li>
-   * <li>Check a single copy of "drill-default.conf". If multiple copies are on the classpath, behavior is
-   * indeterminate.</li>
+   * <li>Check a single copy of "drill-override.conf".  If multiple copies are
+   *     on the classpath, which copy is read is indeterminate.
+   *     If a non-null value for overrideFileResourcePathname is provided, this
+   *     is used instead of "{@code drill-override.conf}".</li>
+   * <li>Check all copies of "{@code drill-module.conf}".  Loading order is
+   *     indeterminate.</li>
+   * <li>Check a single copy of "{@code drill-default.conf}".  If multiple
+   *     copies are on the classpath, which copy is read is indeterminate.</li>
    * </ul>
    *
    * </p>
-   *  @param overrideFileName The name of the file to use for override purposes.
+   * @param overrideFileResourcePathname
+   *          the classpath resource pathname of the file to use for
+   *          configuration override purposes; {@code null} specifies to use the
+   *          default pathname ({@link CommonConstants.CONFIG_OVERRIDE})
+   *          (<strong>not</strong> to not look for an override file at all)
```

Comment:
Sorry for the nitpick, for clarity maybe use something like "<strong>not</strong> to disable looking for an.."